### ISSUE TITLE
2 Heroic launchers. 

### ISSUE THREAD

**Issue by isilkin** [uureel/batocera.pro#30]
Thank you again for a wonderful repo.
If you choose to install heroic launcher from the batocera.pro list everything works as expected. Though, how do I uninstall it? And where is the app folder located for the heroic launcher?
However, if you choose to install lutris it will install every app from baticera.pro. I am not sure if it is the intended behavior. Then installer will give you an option to create an entry in batocera to create game lists from your lutris or heroic for you. So, if you launch heroic from "the games for Linux" entry it is a different version of heroic than the one that installs if you specifically choose heroic in the batocera.pro menu. It also saves game prefixes in a different location and if you choose to create a shortcut nothing happens.
So, to sum it up.
1. Instead of lutris we get every linux app.
2. How to uninstall heroic launcher or any other batocera.pro app.
3. Heroic that comes in the bundle with steam and lutris is not creating game list shortcuts.

To replicate the issue just choose to install lutris from batocera.pro menu.

**Comment by trashbus99** [comment]
1. normal behavior since 90%+ of people use the container for steam.
2. in ~/pro you'll see the apps.  some have uninstall in them.  Otherwise just rm -r ~/pro/<appname>
3.  It won't parse unless you add shortcut in each game in heroic the as the addon message mentons.  The shortcuts appear in ~/Desktop.  Then the parser will parse from those.
